KG - Display SRID When Managing Service Request in Catalog (1.7.0)

kyle-glick opened the pull request.

wtholt commented on app/helpers/application_helper.rb:
```diff
@@ -31,9 +31,13 @@ def show_welcome_message current_user
-  def show_friendly_ssr_id ssr
-    unless ssr.nil?
-      "Editing ID: #{ssr.service_request.protocol.id}-#{ssr.ssr_id}"
+  def protocol_id_display protocol_id, sub_service_request, service_request
```
Per style guide

     def protocol_id_display(protocol_id, sub_service_request, service_request)

jwiel86 commented on Gemfile.lock:
```diff
@@ -165,7 +165,7 @@ GEM
       nokogiri (>= 1.4.3)
     erubis (2.7.0)
-    eventmachine (1.0.8)
+    eventmachine (1.2.0.1)
```
Hey!  Why was this gem updated?

jwiel86 commented on app/helpers/application_helper.rb:
```diff
@@ -31,9 +31,13 @@ def show_welcome_message current_user
     raw(returning_html)
   end
 
-  def show_friendly_ssr_id ssr
```
We're curious, why did you remove this method?

kyle-glick commented:
@jwiel86 
I updated eventmachine because my machine was continuously failing to install the current version we are trying to run. 
The show_friendly_ssr_id method wasn't removed, but rather renamed and modified to fit the new configuration.
